Riot/Modules/Room/RoomViewController.m

```diff
@@ -7871,6 +7855,35 @@ - (void)roomInfoCoordinatorBridgePresenter:(RoomInfoCoordinatorBridgePresenter *
         [[AppDelegate theDelegate] showRoomWithParameters:parameters];
     }
 }
+- (void)roomInfoCoordinatorBridgePresenter:(RoomInfoCoordinatorBridgePresenter *)coordinator
+                       viewEventInTimeline:(MXEvent *)event
+{
+    [self.navigationController popToViewController:self animated:true];
+    [self reloadRoomWihtEventId:event.eventId threadId:event.threadId];
+}
+
+-(void)reloadRoomWihtEventId:(NSString *)eventId
+                  threadId:(NSString *)threadId
```

Comment:
```suggestion
-(void)reloadRoomWihtEventId:(NSString *)eventId threadId:(NSString *)threadId
```
Please put this on one line or just fix the alignment (`:` should align vertically)